int32 sign_label
int32 x
int32 y
int32 width
int32 height
